apollo-router/src/plugins/authorization/mod.rs

```diff
@@ -550,6 +550,8 @@ impl Plugin for AuthorizationPlugin {
         if self.require_authentication {
             ServiceBuilder::new()
                 .checkpoint(move |request: supergraph::Request| {
+                    // XXX(@goto-bus-stop): Why are we doing this here, as opposed to the
```

Comment:
Let's create a ticket for this

Reply:
https://apollographql.atlassian.net/browse/ROUTER-985